Makefile

```diff
@@ -1,6 +1,7 @@
 BUILD_DIR=$(shell pwd)/build
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
 CERT_GEN=$(shell base64 -w 0 scripts/cert-gen.sh)
+VERSION ?= $(shell cat VERSION)
```

Comment:
Any reason you extract the version. I personally see no point here and imo it just blows up the number of files. I am actually going to propose to remove the two version files in the minishift repo after I have found out what purpose they fulfill. Can you give a good reason?